reconf/test_stress_reconf.py

```diff
@@ -0,0 +1,124 @@
+"""
+On the fly reconfiguration stress test.
+"""
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2017-2024 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+from helpers.tf_cfg import cfg
+from reconf.reconf_stress import LiveReconfStressTestBase
+
+SOCKET_START = f"{cfg.get('Tempesta', 'ip')}:443"
+SOCKET_AFTER_RELOAD = f"{cfg.get('Tempesta', 'ip')}:4433"
+STATUS_OK = "200"
+
+TEMPESTA_CONFIG = """
+listen ${tempesta_ip}:443 proto=h2;
+
+tls_certificate ${tempesta_workdir}/tempesta.crt;
+tls_certificate_key ${tempesta_workdir}/tempesta.key;
+tls_match_any_server_name;
+
+cache 0;
+server ${server_ip}:8000;
+"""
+
+
+class TestLiveReconf(LiveReconfStressTestBase):
+    dbg_msg = "Error for curl"
+
+    curl_clients = [
+        {
+            "id": "curl-%s" % count,
+            "type": "external",
+            "binary": "curl",
+            "ssl": True,
+            "cmd_args": ("-Ikf --http2 https://${tempesta_ip}:%s/" % port),
+        }
+        for count, port in enumerate(("443", "4433"))
+    ]
+
+    backends = [
+        {
+            "id": "deproxy",
+            "type": "deproxy",
+            "port": "8000",
+            "response": "static",
+            "response_content": ("HTTP/1.1 200 OK\r\n" "Content-length: 0\r\n" "\r\n"),
+        },
+    ]
+
+    tempesta = {
+        "config": TEMPESTA_CONFIG,
+    }
+
+    def setUp(self) -> None:
+        self.clients.extend(self.curl_clients)
+        super().setUp()
+        self.addCleanup(self.cleanup_clients)
```

Comment:
```suggestion
        super().setUp()
```
`self.addCleanup(self.cleanup_clients)` is called from `super().setUp()`